2021-02-16 15:40:24,534 INFO    MainThread:53555 [wandb_setup.py:_flush():70] setting env: {}
2021-02-16 15:40:24,534 INFO    MainThread:53555 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-02-16 15:40:24,534 INFO    MainThread:53555 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-16 15:40:24,534 INFO    MainThread:53555 [wandb_init.py:_log_setup():319] Logging user logs to /home/naddeok5/FIM/wandb/run-20210216_154024-2mwzfpjb/logs/debug.log
2021-02-16 15:40:24,534 INFO    MainThread:53555 [wandb_init.py:_log_setup():320] Logging internal logs to /home/naddeok5/FIM/wandb/run-20210216_154024-2mwzfpjb/logs/debug-internal.log
2021-02-16 15:40:24,534 INFO    MainThread:53555 [wandb_init.py:init():324] calling init triggers
2021-02-16 15:40:24,534 INFO    MainThread:53555 [wandb_init.py:init():331] wandb.init called with sweep_config: {}
config: {'epochs': 1, 'batch_size': 128, 'momentum': 0.9, 'weight_decay': 0.005, 'learning_rate': 0.01, 'optimizer': 'sgd', 'criterion': 'mse'}
2021-02-16 15:40:24,534 INFO    MainThread:53555 [wandb_init.py:init():401] starting backend
2021-02-16 15:40:24,558 INFO    MainThread:53555 [backend.py:ensure_launched():81] starting backend process...
2021-02-16 15:40:24,578 INFO    MainThread:53555 [backend.py:ensure_launched():86] started backend process with pid: 53958
2021-02-16 15:40:24,579 INFO    MainThread:53555 [wandb_init.py:init():410] backend started and connected
2021-02-16 15:40:24,582 INFO    MainThread:53555 [wandb_init.py:init():433] updated telemetry
2021-02-16 15:40:24,584 INFO    MainThread:53555 [wandb_init.py:init():456] communicating current version
2021-02-16 15:40:25,376 INFO    MainThread:53555 [wandb_init.py:init():461] got version response upgrade_message: "wandb version 0.10.19 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-02-16 15:40:25,377 INFO    MainThread:53555 [wandb_init.py:init():469] communicating run to backend with 30 second timeout
2021-02-16 15:40:25,529 INFO    MainThread:53555 [wandb_init.py:init():492] starting run threads in backend
2021-02-16 15:40:25,609 WARNING MainThread:53555 [wandb_init.py:init():736] interrupted
Traceback (most recent call last):
  File "/home/naddeok5/.local/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 716, in init
    run = wi.init()
  File "/home/naddeok5/.local/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 495, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/naddeok5/.local/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 661, in communicate_run_start
    result = self._communicate(rec)
  File "/home/naddeok5/.local/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 470, in _communicate
    f = future.get(timeout)
  File "/home/naddeok5/.local/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 65, in get
    is_set = self._object_ready.wait(timeout)
  File "/usr/lib/python3.6/threading.py", line 551, in wait
    signaled = self._cond.wait(timeout)
  File "/usr/lib/python3.6/threading.py", line 299, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
